lib/elixir_console/application.ex

```diff
@@ -6,12 +6,17 @@ defmodule ElixirConsole.Application do
   use Application
 
   def start(_type, _args) do
-    # List all child processes to be supervised
     children = [
+      # Start the Telemetry supervisor
+      ElixirConsoleWeb.Telemetry,
```

Comment:
We aren't really doing anything with Telemetry so it is a bit out of place to include it in this PR. What if we remove it and include it later when we decide to add the live dashboard?